session/filesync/filesync.go

```diff
@@ -317,3 +317,15 @@ func CopyFileWriter(ctx context.Context, md map[string]string, c session.Caller)
 
 	return newStreamWriter(cc), nil
 }
+
+type InvalidSessionError struct {
+	Err error
+}
+
+func (e InvalidSessionError) Error() string {
+	return e.Err.Error()
+}
+
+func (e InvalidSessionError) Is(target error) bool {
```

Comment:
Instead of `Is` I guess this should just define `Unwrap` and rest is taken care of by stdlib.